fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraSearch.java

```diff
@@ -87,15 +89,30 @@ public Response doSearch(@QueryParam(value = "condition") final List<String> con
                 final var parsedCondition = parse(condition, identifierConverter());
                 conditionList.add(parsedCondition);
             }
-            final var params = new SearchParameters(conditionList, maxResults, offset);
+
+            List<Condition.Field> parsedFields = null;
+            if (fields == null || fields.trim().length() == 0 || fields.equals("*")) {
```

Comment:
The first 2 clauses could be expressed as `StringUtils.isBlank(fields)`

Reply:
will fix.